examples/fluids/src/setuplibceed.c

```diff
@@ -408,7 +415,7 @@ PetscErrorCode SetupLibceed(Ceed ceed, CeedData ceed_data, DM dm, User user, App
   if (app_ctx->turb_spanstats_enable) PetscCall(TurbulenceStatisticsSetup(ceed, user, ceed_data, problem));
   if (app_ctx->diff_filter_monitor) PetscCall(DifferentialFilterSetup(ceed, user, ceed_data, problem));
 
-  PetscCallCeed(ceed, CeedElemRestrictionDestroy(&elem_restr_jd_i));
+  PetscCallCeed(ceed, CeedElemRestrictionDestroy(&elem_restr_jd_i));  // not used by explicit applications
```

Comment:
Not sure how I feel about adding the comment here. But here's a clarification.

```suggestion
  PetscCallCeed(ceed, CeedElemRestrictionDestroy(&elem_restr_jd_i));  // not used by explicit time integration
```

Reply:
hmmmm   I thought I had removed all of these after using a commit to simply surface the question about whether we wanted to wrap these in a conditional or not.  I will make sure they are fully removed  from this branch as they are not part of the primary objective.

Reply:
Indeed I missed removing this one.  It has been removed now in the latest commit that reverted the StorePackedValues change so I am resolving this.